Library/Formula/saltstack.rb

```diff
@@ -141,5 +150,22 @@ def caveats; <<-EOS.undent
     system "python", "-c", "import M2Crypto"
 
     system "#{bin}/salt", "--version"
+
+    # Requires write access to not be blocked by '--sandbox'
+    # https://github.com/Homebrew/homebrew/pull/44458
+    # assert_no_match /ERROR/, shell_output("#{bin}/salt-call --local grains.ls 2>&1")
+
+    assertDirectory etc/"salt"
+    assertDirectory var/"cache/salt"
+    assertDirectory var/"log/salt"
+    assertDirectory var/"run/salt"
   end
+
+  def assertDirectory(directory)
+
+    assert directory.exist?, "#{directory} must exist"
+    assert directory.directory?, "#{directory} must be a directory"
+    assert directory.writable?, "#{directory} must be writable"
+  end
```

Comment:
Please don’t make a method just for the test. Use a block and loop over directories instead:
```rb
[some_dir1, some_dir2, some_dir3].each do |d|
  assert d.exist?, "..."
  assert d.directory?, "..."
  assert d.writable?, "..."
end
```